backend/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM node:12-alpine
 
 # required for node-gyp
 RUN apk update && apk upgrade && apk add --no-cache --virtual builds-deps build-base \
-  g++ make python tini
+  g++ make python tini python-dev
```

Comment:
I was encountering this build error for workers, and a nearly identical one for backend:
```
Step 4/18 : RUN pip install awscli
 ---> Running in 7e70f6eebdbe
Collecting awscli
  Downloading https://files.pythonhosted.org/packages/69/60/b865ce91eb8965746cc673bc35eac63fe51aed41a757401e1c8bbb6534c9/awscli-1.19.52-py2.py3-none-any.whl (3.6MB)
Collecting colorama<0.4.4,>=0.2.5 (from awscli)
  Downloading https://files.pythonhosted.org/packages/c9/dc/45cdef1b4d119eb96316b3117e6d5708a08029992b2fee2c143c7a0a5cc5/colorama-0.4.3-py2.py3-none-any.whl
Collecting PyYAML<5.5,>=3.10 (from awscli)
  Downloading https://files.pythonhosted.org/packages/a0/a4/d63f2d7597e1a4b55aa3b4d6c5b029991d3b824b5bd331af8d4ab1ed687d/PyYAML-5.4.1.tar.gz (175kB)
  Installing build dependencies: started
  Installing build dependencies: finished with status 'error'
  Complete output from command /usr/bin/python2 -m pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-mdYr_o --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel Cython:
  Collecting setuptools
    Downloading https://files.pythonhosted.org/packages/e1/b7/182161210a13158cd3ccc41ee19aadef54496b74f2817cc147006ec932b4/setuptools-44.1.1-py2.py3-none-any.whl (583kB)
  Collecting wheel
    Downloading https://files.pythonhosted.org/packages/65/63/39d04c74222770ed1589c0eaba06c05891801219272420b40311cd60c880/wheel-0.36.2-py2.py3-none-any.whl
  Collecting Cython
    Downloading https://files.pythonhosted.org/packages/d9/cd/0d2d90b27219c07f68f1c25bcc7b02dd27639d2180add9d4b73e70945869/Cython-0.29.23.tar.gz (2.1MB)
  Installing collected packages: setuptools, wheel, Cython
    Running setup.py install for Cython: started
      Running setup.py install for Cython: finished with status 'error'
      Complete output from command /usr/bin/python2 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-gbx66q/Cython/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-T5pUm8/install-record.txt --single-version-externally-managed --prefix /tmp/pip-build-env-mdYr_o --compile:
      Unable to find pgen, not compiling formal grammar.
      running install
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-2.7
      copying cython.py -> build/lib.linux-x86_64-2.7
      creating build/lib.linux-x86_64-2.7/Cython
      copying Cython/Coverage.py -> build/lib.linux-x86_64-2.7/Cython
      copying Cython/Debugging.py -> build/lib.linux-x86_64-2.7/Cython
      copying Cython/StringIOTree.py -> build/lib.linux-x86_64-2.7/Cython
      copying Cython/Shadow.py -> build/lib.linux-x86_64-2.7/Cython
      copying Cython/CodeWriter.py -> build/lib.linux-x86_64-2.7/Cython
      copying Cython/__init__.py -> build/lib.linux-x86_64-2.7/Cython
      copying Cython/TestUtils.py -> build/lib.linux-x86_64-2.7/Cython
      copying Cython/Utils.py -> build/lib.linux-x86_64-2.7/Cython
      creating build/lib.linux-x86_64-2.7/Cython/Build
      copying Cython/Build/Distutils.py -> build/lib.linux-x86_64-2.7/Cython/Build
      copying Cython/Build/IpythonMagic.py -> build/lib.linux-x86_64-2.7/Cython/Build
      copying Cython/Build/Dependencies.py -> build/lib.linux-x86_64-2.7/Cython/Build
      copying Cython/Build/Cythonize.py -> build/lib.linux-x86_64-2.7/Cython/Build
      copying Cython/Build/BuildExecutable.py -> build/lib.linux-x86_64-2.7/Cython/Build
      copying Cython/Build/Inline.py -> build/lib.linux-x86_64-2.7/Cython/Build
      copying Cython/Build/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Build
      creating build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/CmdLine.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/TreeFragment.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Code.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Symtab.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/CodeGeneration.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/FusedNode.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Annotate.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/ParseTreeTransforms.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/TypeSlots.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Pythran.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Main.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/FlowControl.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Optimize.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Nodes.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Builtin.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/TypeInference.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Visitor.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Scanning.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Lexicon.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Buffer.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Interpreter.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Version.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Errors.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/AnalysedTreeTransforms.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/TreePath.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Options.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/DebugFlags.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/ExprNodes.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Pipeline.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/StringEncoding.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/UtilNodes.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Future.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/PyrexTypes.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/MemoryView.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/ModuleNode.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Naming.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/AutoDocTransforms.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/UtilityCode.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/CythonScope.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Parsing.py -> build/lib.linux-x86_64-2.7/Cython/Compiler
      creating build/lib.linux-x86_64-2.7/Cython/Runtime
      copying Cython/Runtime/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Runtime
      creating build/lib.linux-x86_64-2.7/Cython/Distutils
      copying Cython/Distutils/old_build_ext.py -> build/lib.linux-x86_64-2.7/Cython/Distutils
      copying Cython/Distutils/build_ext.py -> build/lib.linux-x86_64-2.7/Cython/Distutils
      copying Cython/Distutils/extension.py -> build/lib.linux-x86_64-2.7/Cython/Distutils
      copying Cython/Distutils/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Distutils
      creating build/lib.linux-x86_64-2.7/Cython/Debugger
      copying Cython/Debugger/libpython.py -> build/lib.linux-x86_64-2.7/Cython/Debugger
      copying Cython/Debugger/DebugWriter.py -> build/lib.linux-x86_64-2.7/Cython/Debugger
      copying Cython/Debugger/libcython.py -> build/lib.linux-x86_64-2.7/Cython/Debugger
      copying Cython/Debugger/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Debugger
      copying Cython/Debugger/Cygdb.py -> build/lib.linux-x86_64-2.7/Cython/Debugger
      creating build/lib.linux-x86_64-2.7/Cython/Debugger/Tests
      copying Cython/Debugger/Tests/test_libcython_in_gdb.py -> build/lib.linux-x86_64-2.7/Cython/Debugger/Tests
      copying Cython/Debugger/Tests/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Debugger/Tests
      copying Cython/Debugger/Tests/test_libpython_in_gdb.py -> build/lib.linux-x86_64-2.7/Cython/Debugger/Tests
      copying Cython/Debugger/Tests/TestLibCython.py -> build/lib.linux-x86_64-2.7/Cython/Debugger/Tests
      creating build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Machines.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Regexps.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Timing.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Actions.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Traditional.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Lexicons.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Scanners.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Errors.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Transitions.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/DFA.py -> build/lib.linux-x86_64-2.7/Cython/Plex
      creating build/lib.linux-x86_64-2.7/Cython/Tests
      copying Cython/Tests/TestJediTyper.py -> build/lib.linux-x86_64-2.7/Cython/Tests
      copying Cython/Tests/xmlrunner.py -> build/lib.linux-x86_64-2.7/Cython/Tests
      copying Cython/Tests/TestCythonUtils.py -> build/lib.linux-x86_64-2.7/Cython/Tests
      copying Cython/Tests/TestCodeWriter.py -> build/lib.linux-x86_64-2.7/Cython/Tests
      copying Cython/Tests/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Tests
      copying Cython/Tests/TestStringIOTree.py -> build/lib.linux-x86_64-2.7/Cython/Tests
      creating build/lib.linux-x86_64-2.7/Cython/Build/Tests
      copying Cython/Build/Tests/TestStripLiterals.py -> build/lib.linux-x86_64-2.7/Cython/Build/Tests
      copying Cython/Build/Tests/TestIpythonMagic.py -> build/lib.linux-x86_64-2.7/Cython/Build/Tests
      copying Cython/Build/Tests/TestCyCache.py -> build/lib.linux-x86_64-2.7/Cython/Build/Tests
      copying Cython/Build/Tests/TestInline.py -> build/lib.linux-x86_64-2.7/Cython/Build/Tests
      copying Cython/Build/Tests/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Build/Tests
      creating build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestFlowControl.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestMemView.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestTreePath.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestTreeFragment.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestTypes.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestStringEncoding.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestVisitor.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestGrammar.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestSignatureMatching.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestParseTreeTransforms.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestCmdLine.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestUtilityLoad.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      copying Cython/Compiler/Tests/TestBuffer.py -> build/lib.linux-x86_64-2.7/Cython/Compiler/Tests
      creating build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Utility
      creating build/lib.linux-x86_64-2.7/Cython/Tempita
      copying Cython/Tempita/_tempita.py -> build/lib.linux-x86_64-2.7/Cython/Tempita
      copying Cython/Tempita/compat3.py -> build/lib.linux-x86_64-2.7/Cython/Tempita
      copying Cython/Tempita/_looper.py -> build/lib.linux-x86_64-2.7/Cython/Tempita
      copying Cython/Tempita/__init__.py -> build/lib.linux-x86_64-2.7/Cython/Tempita
      creating build/lib.linux-x86_64-2.7/pyximport
      copying pyximport/pyximport.py -> build/lib.linux-x86_64-2.7/pyximport
      copying pyximport/__init__.py -> build/lib.linux-x86_64-2.7/pyximport
      copying pyximport/pyxbuild.py -> build/lib.linux-x86_64-2.7/pyximport
      creating build/lib.linux-x86_64-2.7/Cython/Includes
      copying Cython/Includes/openmp.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes
      creating build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/iterator.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/vector.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/functional.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/complex.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/forward_list.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/unordered_set.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/algorithm.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/string.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/typeinfo.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/set.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/cast.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/memory.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/deque.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/utility.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/limits.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/stack.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/typeindex.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/unordered_map.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/queue.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/list.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/pair.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/__init__.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      copying Cython/Includes/libcpp/map.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libcpp
      creating build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/resource.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/wait.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/ioctl.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/mman.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/dlfcn.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/types.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/time.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/stat.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/signal.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/select.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/fcntl.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/strings.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/__init__.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/unistd.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/stdlib.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      copying Cython/Includes/posix/stdio.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/posix
      creating build/lib.linux-x86_64-2.7/Cython/Includes/numpy
      copying Cython/Includes/numpy/math.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/numpy
      copying Cython/Includes/numpy/__init__.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/numpy
      creating build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_exc.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_version.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_type.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_method.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_oldbuffer.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_string.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_module.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_set.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_long.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_weakref.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_ref.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_unicode.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_function.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_number.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_instance.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_buffer.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_sequence.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_cobject.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_int.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_bytes.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_complex.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_pycapsule.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_tuple.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_object.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_float.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_iterator.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_bool.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_dict.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_mapping.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_getargs.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/stdlib.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_mem.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/stdio.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/python_list.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      copying Cython/Includes/Deprecated/stl.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/Deprecated
      creating build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/errno.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/setjmp.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/string.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/time.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/locale.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/float.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/signal.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/limits.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/math.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/stdint.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/stddef.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/__init__.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/stdlib.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      copying Cython/Includes/libc/stdio.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/libc
      creating build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/iterator.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/exc.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/number.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/slice.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/int.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/method.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/type.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/complex.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/conversion.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/pystate.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/bool.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/string.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/iterobject.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/unicode.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/long.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/set.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/version.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/instance.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/pythread.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/mapping.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/memoryview.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/float.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/oldbuffer.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/sequence.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/codecs.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/bytes.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/mem.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/bytearray.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/ceval.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/object.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/longintrepr.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/cellobject.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/weakref.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/cobject.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/dict.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/module.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/function.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/list.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/ref.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/getargs.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/pylifecycle.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/pycapsule.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/tuple.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/array.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/datetime.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/__init__.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/genobject.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Includes/cpython/buffer.pxd -> build/lib.linux-x86_64-2.7/Cython/Includes/cpython
      copying Cython/Compiler/Visitor.pxd -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Parsing.pxd -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Scanning.pxd -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/Code.pxd -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/FlowControl.pxd -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Compiler/ParseTreeTransforms.pxd -> build/lib.linux-x86_64-2.7/Cython/Compiler
      copying Cython/Runtime/refnanny.pyx -> build/lib.linux-x86_64-2.7/Cython/Runtime
      copying Cython/Debugger/Tests/codefile -> build/lib.linux-x86_64-2.7/Cython/Debugger/Tests
      copying Cython/Debugger/Tests/cfuncs.c -> build/lib.linux-x86_64-2.7/Cython/Debugger/Tests
      copying Cython/Plex/Actions.pxd -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Plex/Scanners.pxd -> build/lib.linux-x86_64-2.7/Cython/Plex
      copying Cython/Utility/TestCyUtilityLoader.pyx -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/CConvert.pyx -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/MemoryView.pyx -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/CppConvert.pyx -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/TestCythonScope.pyx -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/CpdefEnums.pyx -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/TestUtilityLoader.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/MemoryView_C.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Coroutine.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/CythonFunction.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/AsyncGen.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Printing.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Embed.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/StringTools.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Builtins.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/ImportExport.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Complex.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Exceptions.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Optimize.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/ObjectHandling.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/CMath.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/CommonStructures.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/TypeConversion.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/FunctionArguments.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Profile.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/ModuleSetupCode.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Overflow.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/ExtensionTypes.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Capsule.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/Buffer.c -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/arrayarray.h -> build/lib.linux-x86_64-2.7/Cython/Utility
      copying Cython/Utility/CppSupport.cpp -> build/lib.linux-x86_64-2.7/Cython/Utility
      running build_ext
      building 'Cython.Plex.Scanners' extension
      creating build/temp.linux-x86_64-2.7
      creating build/temp.linux-x86_64-2.7/tmp
      creating build/temp.linux-x86_64-2.7/tmp/pip-install-gbx66q
      creating build/temp.linux-x86_64-2.7/tmp/pip-install-gbx66q/Cython
      creating build/temp.linux-x86_64-2.7/tmp/pip-install-gbx66q/Cython/Cython
      creating build/temp.linux-x86_64-2.7/tmp/pip-install-gbx66q/Cython/Cython/Plex
      gcc -fno-strict-aliasing -Os -fomit-frame-pointer -g -DNDEBUG -Os -fomit-frame-pointer -g -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/usr/include/python2.7 -c /tmp/pip-install-gbx66q/Cython/Cython/Plex/Scanners.c -o build/temp.linux-x86_64-2.7/tmp/pip-install-gbx66q/Cython/Cython/Plex/Scanners.o
      /tmp/pip-install-gbx66q/Cython/Cython/Plex/Scanners.c:21:10: fatal error: Python.h: No such file or directory
         21 | #include "Python.h"
            |          ^~~~~~~~~~
      compilation terminated.
      error: command 'gcc' failed with exit status 1
  
      ----------------------------------------
  Command "/usr/bin/python2 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-gbx66q/Cython/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-T5pUm8/install-record.txt --single-version-externally-managed --prefix /tmp/pip-build-env-mdYr_o --compile" failed with error code 1 in /tmp/pip-install-gbx66q/Cython/
  
  ----------------------------------------
Command "/usr/bin/python2 -m pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-mdYr_o --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel Cython" failed with error code 1 in None
The command '/bin/sh -c pip install awscli' returned a non-zero code: 1
```

Reply:
Were these errors only recently introduced in this PR?

Reply:
I'm not sure. It didn't occur when I was testing it yesterday but suddenly occured when I reran a job today (even without changing any code).

Reply:
I'll try looking further into it.

Reply:
Seems like an issue that occured with installation of `aws-cli`. Maybe we didn't hit this issue previously as this layer in the Docker image is cached.

Reply:
Yeah, maybe an update to `aws-cli` caused it? I think there isn't any issue with installing `python-dev`.